service/src/main/java/bio/terra/workspace/service/petserviceaccount/PetSaService.java

```diff
@@ -54,35 +53,49 @@ public PetSaService(
    * Wrapper around {@code enablePetServiceAccountImpersonationWithEtag} without requiring a
    * particular GCP eTag value.
    */
-  public Policy enablePetServiceAccountImpersonation(UUID workspaceId, UserWithPetSa userAndPet) {
+  public Policy enablePetServiceAccountImpersonation(
+      UUID workspaceId, AuthenticatedUserRequest userRequest) {
     // enablePetServiceAccountImpersonationWithEtag will only return an empty optional if the
     // provided eTag does not match current policy. Because we do not use eTag checking here, this
     // is always nonempty.
-    return enablePetServiceAccountImpersonationWithEtag(workspaceId, userAndPet, null).get();
+    return enablePetServiceAccountImpersonationWithEtag(workspaceId, userRequest, null).get();
```

Comment:
I'd use `orElseThrow()` instead of `get()` so you get a more meaningful error message when it's not found.

Reply:
Done